content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -1,8 +1,17 @@
 # Kubernetes
 
-## Description
+Deploy a model to a kubernetes cluster exposing its prediction endpoints through
+a service.
+
+## Preparation
 
-**TODO**
+- Make sure you have a Kubernetes cluster accessible, with the corresponding
+  kubeconfig file available.
+- The cluster has access to a docker registry so as to pull docker images.
+- Relevant permissions to create resources on the cluster -- deployment,
+  service, etc. are present.
+- Nodes are accessible and reachable, with an external IP address (valid for a
+  NodePort service, more details to come below).
```

Comment:
I would link k8s docs pages here

Reply:
I have added a link to a basic tutorial here: https://kubernetes.io/docs/tutorials/kubernetes-basics/